introduction.md

```diff
@@ -33,48 +34,50 @@ The Task Force found that the majority of success criteria from WCAG 2 can apply
-*   References to glossary items in this document are presented in `<cite>` elements visually styled as ordinary text with a dark gray underline.
-*   Hereafter, the short title “WCAG2ICT” is used to reference this document.
-*   In headings the term "Success Criterion" has been shortened to “SC” for brevity.
+* Quotes from WCAG 2 and WCAG 2 Understanding Documents are in `<blockquote>` elements and visually styled with a gray bar on the left, and immediately follow the heading for the principle, guideline, or success criterion.
+* Additional guidance provided by this document begins with the phrase “Applying” and has no special visual styling.
+* Replacement text that is presented to show how an SC would read as modified by the advice in this document are in `<ins>` elements visually styled as bold green text with a dotted underline.
+* Notes are slightly inset and begin with the phrase “NOTE”. Each note is in its own inset box styled in pale green with a darker green line on the left side of the box.
+* References to glossary items from WCAG 2 are presented in `<cite>` elements visually styled as ordinary text with a dotted underline, and contain title attributes noting these are WCAG definitions.  They turn blue with a yellow background when mouse or keyboard focus is placed over them.
+* References to glossary items in this document are presented in `<cite>` elements visually styled as ordinary text with a dark gray underline.
+* Hereafter, the short title “WCAG2ICT” is used to reference this document.
+* In headings the term "Success Criterion" has been shortened to “SC” for brevity.
 
 ### Comparison with the 2013 WCAG2ICT Note
 
-<p class="ednote">The WCAG2ICT Task Force has incorporated all of the new WCAG 2.1 guidelines, criteria and glossary terms. The next draft version will incorporate new WCAG 2.2 criteria and glossary terms as well as address open issues on any of the content in the document.</p>
+<div class="ednote"><p>The WCAG2ICT Task Force has incorporated all of the new WCAG 2.1 guidelines, criteria and glossary terms. The next draft version will incorporate new WCAG 2.2 criteria and glossary terms as well as address open issues on any of the content in the document.</p>
 
 The following changes and additions have been made to update the 2013 WCAG2ICT document to incorporate the [new features in WCAG 2.1](https://www.w3.org/TR/WCAG21/#new-features-in-wcag-2-1), the [new features in WCAG 2.2](https://www.w3.org/TR/WCAG22/#new-features-in-wcag-2-2), and the change to 4.1.1 Parsing listed in the [Comparison with WCAG 2.1](https://www.w3.org/TR/WCAG22/#comparison-with-wcag-2-1) section:
 
-* New [Background](#background) section to explain the history and known uses of WCAG2ICT
-* New WCAG 2.1 Success Criteria and Guidelines
+- New [Background](#background) section to explain the history and known uses of WCAG2ICT
```

Comment:
These notation changes weren't needed (replacing "*" with "-". Since this is markdown, it is converted into an unordered list.

Reply:
I used a difference character for 2nd level UL LI so that easier to spot when using screen reading software.